### DISCUSSION TITLE
Serveur python

### DISCUSSION THREAD

**Discussion by MKH67** [pifou25/jeedom-lgthinq-plugin#7] [Q&A]
hello !

merci pour ce plugin.

J'ai un peu de mal, autant pour la partie jeedom tout est ok, autant pour python et docker, je ne sais pas par ou commencer.

est-il possible d'etre un peu plus explicite dans la doc ou que l'on se contacte ?

merci en tout cas !

**Comment by pifou25** [comment]
Bonjour,

Je peux être plus explicite dans ma doc, c'est toujours une bonne idée :) 
J'étais débutant sur python et docker lorsque j'ai commencé ce projet l'an dernier, donc tout n'est pas forcément optimum, ni conventionnel...

En gros, on a une lib python (wideq) sur laquelle j'essaye de faire une API HTTP (donc python aussi) pour intégrer le tout dans mon plugin jeedom.
docker n'est pas indispensable, je l'utilise pour mes tests mais pas dans le plugin final. Oublie pour l'instant, et concentre tes questions sur la partie python si besoin :)

**Comment by MKH67** [comment]
Bonjour 

Merci pour ce retour

Alors déjà avec les différentes updates, j'ai réussi à synchroniser les matériels et remonter les informations :) 

Donc de mon côté, plus rien à redire ^^
